tests/cmdline/commands/test_status.py

```diff
@@ -83,6 +84,7 @@ def storage_cls(*args, **kwargs):
     assert result.exit_code is ExitCode.CRITICAL
 
 
+@pytest.mark.requires_psql
```

Comment:
Why is this mark required here now?

Reply:
Because it is explicitly using the `PsqlDosMigrator`, so that will fail if the test profile is run with a `core.sqlite_dos` profile, i.e. with the `presto` marker